azure-pipelines.yml

```diff
@@ -55,6 +56,8 @@ jobs:
     displayName: 'Install nni toolkit via source code'
   - script: |
       cd test
+      ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" < /dev/null 2> /dev/null
```

Comment:
why we need this?

Reply:
I'm still having a hard time to pass the UT. I'll notice everybody for review once this PR is actually ready...

Reply:
OK

Reply:
Dependency chain: SMAC -> swig -> brew